utils/build/docker/ruby/sinatra14/app.rb

```diff
@@ -13,9 +14,7 @@
 Datadog.configure do |c|
   c.diagnostics.debug = true
 
-  unless c.respond_to?(:tracing)
-    c.use :sinatra, service_name: (ENV['DD_SERVICE'] || 'sinatra')
-  end
+  c.use :sinatra, service_name: ENV['DD_SERVICE'] || 'sinatra' unless c.respond_to?(:tracing)
```

Comment:
```suggestion
  c.use :sinatra, service_name: ENV.fetch('DD_SERVICE', 'sinatra') unless c.respond_to?(:tracing)
```

Reply:
not sure I touched this line, but it makes sense to fix it of course

Reply:
oh, it seems that my editor automatically fixed all linter warnings